### PR TITLE
Added the resend activation e-mail endpoint

### PR DESCRIPTION
**Pull request by zefciu**
The resend activation e-mail view that was provided by @sargo was added with documentation and tests.

Fixes: #168

### CODE REVIEW

File: djoser/urls/base.py [hunk 1]
```diff
@@ -27,6 +27,11 @@
         views.ActivationView.as_view(),
         name='user-activate'
     ),
+    url(
+        r'^users/resend/?$',
+        views.ResendActivationView.as_view(),
+        name='user-activate'
```
[the remark this answers is not in the document]
Comment (khashashin) [reply djoser/urls/base.py]: > @zefciu It has the same name as `ActivationView` (`user-activate`) ;-)

is it better here to use the same name? Its overwrites the activation url.

[the remark this answers is not in the document]
Comment (zefciu) [reply djoser/urls/base.py]: Thanks for noticing it

**Comment by alexandernst** [comment]
Is this good to get merged? @dekoza

File: docs/source/base_endpoints.rst [hunk 2]
```diff
@@ -98,6 +98,23 @@ request to activate endpoint.
...
+be sent if the user is already active or if they don't have a usable password.
+Also if the sending of activation e-mails is disabled in settings, this call
+will result in ``HTTP_400_BAD_REQUEST``
+
+**Default URL**: ``/users/resend/``
+
++----------+----------------+----------------------------------+
+| Method   | Request        | Response                         |
++==========+================+==================================+
+| ``POST`` | * ``uid``      | ``HTTP_204_NO_CONTENT``          |
```
Comment (dekoza) [inline docs/source/base_endpoints.rst]: Shouldn't it be `email`?

**Pull request merged**